src/lib.rs

```diff
@@ -941,6 +941,23 @@ fn test_is_on_curve_var() {
     assert!(AffinePoint::identity().is_on_curve_vartime());
 }
 
+#[test]
+fn test_affine_point_generator() {
+    AffinePoint::generator();
+}
+
```

Comment:
This is not a test. This is just calling a function inside of a tests. Makes no sense to do that.

Either you do something with the generator or you can remove the tests

Reply:
This is now removed.